sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncRepository.kt

```diff
@@ -33,6 +33,7 @@ import timber.log.Timber
 
 interface SyncRepository {
     fun createAccount(): Result<Boolean>
+    fun isSignedIn(): Boolean
```

Comment:
nit:  documentation

Reply:
no need, this is still inside `impl` module.